src/vm-memory/src/bitmap.rs

```diff
@@ -78,6 +78,13 @@ impl Bitmap {
     pub fn is_empty(&self) -> bool {
         false
     }
+
+    /// Reset all bitmap bits to 0.
+    pub fn reset(&self) {
+        for bit in self.map.iter() {
+            bit.store(0, Ordering::Release);
```

Comment:
nit: rename `bit` to `element` or `it` or smth else since each loop we reset 64bits, not just one. The current name is misleading.